### ISSUE TITLE
Increase Team Size 

### ISSUE THREAD

**Issue by ajmyers01** [TarkovTracker/TarkovTracker#114]
I noticed this seems to be configured in the firebase databse. https://github.com/TarkovTracker/TarkovTracker/blob/master/src/views/Settings.vue#L149

Is there any chance we can increase the number of team members above 5, or is there a technological limitation.

```
Scenario we have 6 friends who play tarkov together and use the tracker to figure out what smaller teams to break into to complete quests. 

Problem: Tarkov tracker won't allow us to have 6 people to a "team" 

Proposed Solution: Increase the variable firesys/team@maximumMembers to 6 or higher.
 If there is not a reason for the 5 man limit, other than ingame party size limit.
```

**Comment by thaddeus** [comment]
Its set at five for a couple of reasons, one is that is the maximum party size in game, two is that since each person in a team gets all the live updates from their teammates, reads increase exponentially with team size. I was playing it safe, planning for the worst case scenario of usage.

That being said, it should be totally fine to increase to 10 generally, I just haven't gotten to it. Having an issue to remind myself will be good, and should get this done in the next day or two.

> Scenario we have 6 friends who play tarkov together and use the tracker to figure out what smaller teams to break into to complete quests. 

That is one of my goals with #11 - I want groups up to something like 50 people be able to find people within the group who have X number of same/similar quests based upon specific objectives, or generally maps. I'd have a cached endpoint per team for something like ~5 minute periods to limit large batches of reads, but allow larger groups to split up easier.

Will definitely increase the team max for now though, thanks for the reminder!

**Comment by thaddeus** [comment]
Fixed in #123
